modules/nlu/src/backend/pipelines/entities/pattern_extractor.ts

```diff
@@ -144,6 +152,14 @@ export default class PatternExtractor {
     return findings
   }
 
+  private _is(limits: limits) {
```

Comment:
I would not create such an helper object for that small task.

Also, you basically just want to check that both your start, end is inside match.meta, _.inRange almost already give you that. I would go for something like this;
*** at the top of file
//import {inRange} from 'lodash/fp'
const  inBoundedRange = ({start, end}) => (val) => _.inRange(start, end)
***
later in your code...
***

conts isInMetaRange = inBoundedRange(match.meta)
[start, end].every(isInMetaRange)

Reply:
also, we acutally have a allInRange function in `modules/nlu/backend/tools/math.ts`